NOTICE

```diff
@@ -0,0 +1,28 @@
+j2objc-common-libs-e2e-test - https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test
+  a subproject of j2objc-gradle - Gradle plugin for j2objc (Java to Objective-C transpiler)
+  https://github.com/j2objc-contrib/j2objc-gradle
+
+Copyright (c) 2015 the authors of j2objc-gradle
+
+Main Contributors:
+  Advay Mengle @advayDev1 <source@madvay.com>
+  Bruno Bowden @brunobowden <github@brunobowden.com>
+  Michael Gorski @confile <mail@michaelgorski.de>
+  
+Thanks:
+  Peter Niederweiser @pniederw <peter@pniederw.com>
+  Sterling Greene @big-guy <sterling.greene@gradleware.com>
```

Comment:
hmm, ok then I changed it to:
`Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test`
instead of j2objc-gradle.

which means I can only put my name for the moment.  Please add your name in your own commit.